domain/consensus/processes/ghostdagmanager/ghostdag_test.go

```diff
@@ -177,6 +178,91 @@ func TestGHOSTDAG(t *testing.T) {
 	})
 }
 
+func TestBlueWork(t *testing.T) {
```

Comment:
Please add a rudimentary drawing of the DAG you're building in this test, as well as an explanation as to what this test tests.